lib/getLeoConfigFromBusStack.js

```diff
@@ -0,0 +1,33 @@
+const AWS = require('aws-sdk');
+const logger = require('leo-logger');
+
+module.exports = async function (leoBusStackName, credentials) { // AWS.Credentails
+	var cloudformation = (credentials)
+		? new AWS.CloudFormation({ credentials })
+		: new AWS.CloudFormation();
+
+	var params = { StackName: leoBusStackName };
+	const descStackresult = await cloudformation.describeStacks(params).promise();
+	if (descStackresult.Stacks.length > 1) {
+		logger.info(descStackresult.Stacks);
+		throw new Error('Multiple stacks match criteria');
+	}
+	const stackOutputs = descStackresult.Stacks[0].Outputs;
```

Comment:
This would prevent calling `.find` over and over
`const stackOutputs = descStackresult.Stacks[0].Outputs.reduce((map, output)=>{
	map[output.OutputKey] = output.OutputValue;
	return map;
}, {});
{ LeoCron: stackOutputs.LeoCron }`